src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$color-main-accent: #060b35;
+$color-secondary: #616070;
+$color-accent-blue: #00acdc;
+$color-border-card: #f3f3f3;
+$img-product-height: 134px;
+$img-product-width: 160px;
+$img-star-height-width: 16px;
```

Comment:
Checklist point 11

Create variables only for values that are being used more than once